Tests/Acceptance/ID3TagEditorTestAcceptanceNewFrames.swift

```diff
@@ -42,13 +45,16 @@ class ID3TagEditorTestAcceptanceNewFrames: XCTestCase {
         XCTAssertEqual(id3Tag?.frames[.Subtitle]?.id3Identifier, "TT3")
         XCTAssertEqual((id3Tag?.frames[.Subtitle] as? ID3FrameWithStringContent)?.content, "Subtitle V2")
         XCTAssertEqual(id3Tag?.frames[.UnsyncedLyrics]?.id3Identifier, "ULT")
+        #warning("The data in the “ULT” tag is “55 4C 54 00 00 05 00 00 10 00 00”. That appears to mean “ULT”, a 5‐byte size, a text encoding of Latin‐1, a language of “null, data escape, null”, and nothing more. But that is nonsense. The expected string is not present. What is actually in the file?")
         XCTAssertEqual((id3Tag?.frames[.UnsyncedLyrics] as? ID3FrameCommentTypes)?.content, "UnsyncedLyrics V2")
         XCTAssertEqual((id3Tag?.frames[.UnsyncedLyrics] as? ID3FrameCommentTypes)?.language, .und)
         XCTAssertEqual((id3Tag?.frames[.UnsyncedLyrics] as? ID3FrameCommentTypes)?.description, "lyricsTest V2")
+        #warning("COM is never detected. However, the garbled “ULT” tag is immediately followed by a “#[null]u” tag which gets discarded as invalid. The data that frame reports is a whopping 158206 bytes, and starting from the “u” in the tag name, it begins with “und”, two nulls and then “lyricsTest V2”. It seems the previous tag extends beyond its own declared end. Either the file was encoded wrong or I am working from the wrong specification. Either way, the root of the problem lies at the point when the frames themselves are split up, before individual frames are interpreted.")
```

Comment:
Yeah, I've got nothing. I knew what I'd come up with wasn't going to be compliant, but I couldn't figure out how to fix it.

Aside from the frame identifier and version, there are three parameters for ULT(USLT) and COM(COMM): the 3-letter language code, which should be invisible, the description, which should be a terminated string, and the content itself. Either I got the language part wrong, or the terminator part, or both.

Pretty much any frame with a description string can have multiple versions of that frame in the tag as long as the language and description are differnt. I.E:

frames: [
.UnsyncedLyrics : ID3FrameCommentTypes(language: .eng, description: "EnglishLyrics", content: "These are the lyrics in English"),
.UnsyncedLyrics : ID3FrameCommentTypes(language: .fra, description: "FrenchLyrics", content: "These are the lyrics in French"),
]